rpcs3/Emu/Cell/Modules/cellGame.cpp

```diff
@@ -848,7 +865,15 @@ error_code cellGameGetSizeKB(vm::ptr<s32> size)
 		}
 	}
 
-	*size = ::narrow<u32>(fs::get_dir_size(local_dir) / 1024);
+	idm::select<lv2_fs_object, lv2_file>([&](u32, lv2_file& file)
```

Comment:
Better fo compiler optimzations
```suggestion
	idm::select<lv2_fs_object, lv2_file>([](u32, lv2_file& file)
```